junit-jupiter-api/src/main/kotlin/org/junit/jupiter/api/Assertions.kt

```diff
@@ -126,12 +125,9 @@ inline fun <reified T : Throwable> assertThrows(message: String, executable: ()
  * @see Assertions.assertThrows
  */
 inline fun <reified T : Throwable> assertThrows(noinline message: () -> String, executable: () -> Unit): T {
-    val result: Result<*> = runCatching(executable)
-    val executable = Executable {
-        result.getOrThrow()
-    }
+    val result = runCatching(executable)
 
-    return Assertions.assertThrows(T::class.java, executable, Supplier(message))
+    return Assertions.assertThrows(T::class.java, Executable { executable }, Supplier(message))
```

Comment:
`executable` should be `result`, doesn't compile

Reply:
🤦‍♂